test/views/messages/ReportingFrequencyViewMessages.scala

```diff
@@ -31,15 +37,34 @@ object ReportingFrequencyViewMessages {
-  val mandatoryReportingInset = "Reporting quarterly is required if your income from self-employment or property, or both, exceeds the income threshold."
-  val mandatoryReportingText = "You can check the threshold for when reporting quarterly is mandatory in the criteria for people who will need to sign up for Making Tax Digital for Income Tax (opens in new tab)"
-  val mandatoryReportingLink = "criteria for people who will need to sign up for Making Tax Digital for Income Tax (opens in new tab)"
+  val differentObligationsHeading = "What the different reporting obligations are"
+  val differentObligationsText = "When you are using Making Tax Digital for Income Tax you need to submit:"
+  val differentObligationsLiOne = "4 updates a year for each source of self employment and property income"
+  val differentObligationsLiTwo = "a tax return"
+  val differentObligationsTextTwo = "For tax years you are opted out of, you only need to file a tax return."
+  val differentObligationsTextThree = "You can see what is due on the updates and deadlines page."
+
+  def mandatoryReportingHeading(isNewContentEnabled: Boolean): String =
+    if(isNewContentEnabled) "When Making Tax Digital for Income Tax is required"
+    else "When reporting quarterly is mandatory"
+  def mandatoryReportingInset(isNewContentEnabled: Boolean): String =
+    if(isNewContentEnabled) "You must use Making Tax Digital for Income Tax if your total gross income from self-employment or property, or both, exceeds the £50,000 threshold."
+    else "Reporting quarterly is required if your income from self-employment or property, or both, exceeds the income threshold."
+  def mandatoryReportingText(isNewContentEnabled: Boolean): String =
+    if(isNewContentEnabled) "If this happens in the 2022 to 2023 tax year, you would be required to use Making Tax Digital for Income Tax for the 2024 to 2025 tax year."
+    else "You can check the threshold for when reporting quarterly is mandatory in the criteria for people who will need to sign up for Making Tax Digital for Income Tax (opens in new tab)"
+  val newMandatoryReportingTextTwo = "Find out more about the threshold in the criteria for people who will need to sign up for Making Tax Digital for Income Tax (opens in new tab)"
+
+  def compatibleSoftwareHeading(isNewContentEnabled: Boolean): String =
+    if(isNewContentEnabled) "When you need compatible software"
+    else "When compatible software is required"
 
-  val compatibleSoftwareH2 = "When compatible software is required"
-  val compatibleSoftwareP1 = "If you are reporting quarterly, you will need to use software compatible with Making Tax Digital for Income Tax (opens in new tab) There are both paid and free options for you or your agent to choose from."
-  val compatibleSoftwareP2 = "For reporting annually, you can use either compatible software or your HMRC online account to submit your tax return."
+  def compatibleSoftwareText(isNewContentEnabled: Boolean): String =
```

Comment:
isNewContentEnabled this can be renamed to something like optInOptOutContentUpdateR17IsEnabled just so we know what's driving the logic.